collection-db-connect.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+const HOST = 'db';
+const USER = 'root';
+const password = 'password';
```

Comment:
Change this const name to be all uppercase - we always give consts uppercase names just to make it really obvious that its a const when it gets used.